aries-site/src/examples/templates/feedback/PowerOnExample.js

```diff
@@ -0,0 +1,205 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import {
+  Anchor,
+  Box,
+  Button,
+  Card,
+  Layer,
+  Form,
+  Footer,
+  FormField,
+  Heading,
+  Spinner,
+  StarRating,
+  ResponsiveContext,
+  Text,
+  TextArea,
+} from 'grommet';
+import { StatusGoodSmall } from 'grommet-icons';
+
+const FeedbackLayout = ({ children, onClick, onClose }) => {
+  const breakpoint = useContext(ResponsiveContext);
+  return (
+    // feedback layout
+    <Layer>
+      <Box
+        fill="vertical"
+        overflow="auto"
+        width={!['xsmall', 'small'].includes(breakpoint) ? 'medium' : undefined}
```

Comment:
Let's connect on this, we shouldn't need to specify a width. Instead, we should be using a [content-driven layout](https://design-system.hpe.design/templates/content-layouts?q=content-d#content-driven-layouts).